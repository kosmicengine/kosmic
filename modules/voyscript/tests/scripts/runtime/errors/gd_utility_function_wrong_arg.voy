func test():
	var x = Color()
	print(len(x)) # VoyScript utility function.
