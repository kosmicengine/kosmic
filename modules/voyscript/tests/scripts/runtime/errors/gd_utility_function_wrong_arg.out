VOYTEST_RUNTIME_ERROR
>> SCRIPT ERROR at runtime/errors/ks_utility_function_wrong_arg.voy:3 on test(): Error calling VoyScript utility function "len()": Value of type 'Color' can't provide a length.
